azure/build-pipeline.yml

```diff
@@ -34,7 +34,7 @@ jobs:
           jdkArchitectureOption: 'x64'
           publishJUnitResults: true
           testResultsFiles: '**/TEST-*.xml'
-          tasks: 'publishLinuxMips32PublicationToMavenLocal' # No test target, but this will ensure it builds
+          tasks: 'build'
```

Comment:
@russhwolf is there a good reason to do the publish rather than build?

Reply:
If you run build on every platform, you duplicate work. ie JVM will be built three times. My original config was trying to make the macOS build do most of the work, and the other two build only the things that macOS can't. For MingwX64 you have a native test target to run, and that will ensure the correct build task runs as well. For Mips there's no test binary generated so need to do something else to force a build. Could pull out the correct compile/link tasks instead of doing publishToMavenLocal but in any case, the intent was to build only that binary and nothing else.